object-details/design.md

```diff
@@ -0,0 +1,29 @@
+# Object details
+
+
+## Sidecar field
+![](img/sidecar-suggestion.png)
+- We could add a `sidecar` field to the item details to illustrate whether the sidecar is missing or represent
+
+## Minor page structure improvements
+These changes should be relatively simple to make, but improve the information hierarchy of object details pages.
+
+![](img/minimal-changes.png)
+- Move page title and refresh button to above tabs
+- Remove extraneous items from breadcrumbs
+- Remove card border from object details
+
+![](img/minimal-changes-alt.png)
+- This structure would make it more appropriate to include states like "missing sidecar" in the header of the page
+
+## Moderate style changes
+Adds more space to the page and brings it in line with other cloud software
+
+![](img/moderate-changes.png)
+- Change font and table styling
+- Move health into header (details still available on hover)
+- More space for ports and endpoints lists
+
+## Big changes big difference
```

Comment:
this is great as well, I think it should be a different item. As I discussed with @serenamarie125 I'm in favor of this, I'd like to understand better how long does it take to convert pf3 to pf4. Also worth to understand - why pf3 is not getting this style change so all projects can enjoy it "for free"? With the current situation, every project has to do separate work to make it look like this.

Reply:
@abonas we need to set up a meeting with the appropriate PatternFly team members.  Who are the appropriate people to include on the Service Mesh side?

Reply:
> @abonas we need to set up a meeting with the appropriate PatternFly team members. Who are the appropriate people to include on the Service Mesh side?

it depends what is the goal of the meeting - can you elaborate?